build/csharpTranspiler.ts

```diff
@@ -330,6 +330,7 @@ class NewTranspiler {
         }
 
         const csharpReplacements = {
+            'DictOrDictList': 'List<Dictionary<string, T>>',
```

Comment:
A comment on this [stackOverflow post](https://stackoverflow.com/questions/78282588/c-how-to-create-a-custom-type-that-can-be-a-dictionary-or-a-list-of-dictionari) said that this would work

It doesn't look like the transpiler is working though?

Reply:
this type does not make sense, have you tested it?

Reply:
> this type does not make sense, have you tested it?

I updated the `T` to `object`, but it doesn't get transpiled anyway, so running `npm run test` won't test it out, if I insert it in manually no errors show up in my vscode

Reply:
@carlosmiei Are you able to see why the type is not being transpiled? Into C# and Python?